tests/unittest/_torch/executor/test_py_scheduler.py

```
@@ -3079,6 +3079,137 @@ def test_no_reuse_when_disabled(self):
+            max_num_requests=4,
+            kv_cache_manager=kv,
+            peft_cache_manager=peft,
+            scheduler_policy=CapacitySchedulerPolicy.GUARANTEED_NO_EVICT,
+        )
+        tokens = list(range(21))
+        # Same prompt and same LoRA task id => same block key, so r1 is a
+        # duplicate of r0. r0 demands more PEFT pages than are available.
+        r0 = _make_request(0, prompt_len=21, input_tokens=tokens, lora_task_id=7)
+        r1 = _make_request(1, prompt_len=21, input_tokens=tokens, lora_task_id=7)
+
+        fitting, disagg, paused = scheduler.schedule_request([r0, r1])
```

Ruff (0.16.2)

[warning] Unpacked variable `disagg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)